apps/condo/domains/marketplace/schema/GetInvoicesWithSkuInfoService.js

```diff
@@ -40,7 +40,7 @@ const GetInvoicesWithSkuInfoService = new GQLCustomSchema('GetInvoicesWithSkuInf
                 const where = {
                     deletedAt: null,
                     client: { id: userId },
-                    context: { organization: { id: organizationId } },
+                    organization: { id: organizationId },
```

Comment:
`deletedAt` or not?

Reply:
Asked the developer. But invoices can't be created without context & organization.

Reply:
We decided to remove this query. In the next PR.